### PR TITLE
MRG: switch to AGPL license

### PR DESCRIPTION
**Pull request by ctb**
as discussed on slack and in https://github.com/sourmash-bio/pyo3_branchwater/issues/60, we are switching the license over to AGPL from BSD.

### CODE REVIEW

**Comment by ctb** [comment]
@mr-eyes @bluegenes can you thumbs up or approve? thx.

**Comment by mr-eyes** [comment]
I don't have review/write permissions 😅

**Comment by ctb** [comment]
(or not, if you don't...)

File: README.md [hunk 1]
```diff
@@ -121,4 +121,13 @@ maturin build
...
+## License
+
+This software is under the AGPL license. Please see [LICENSE.txt](LICENSE.txt).
+
+## Authors
+
+Luiz Irber
+C. Titus Brown
+Mo Abuelanin
+N. Tessa Pierce
```
Comment (bluegenes) [inline README.md:133]:
```suggestion
N. Tessa Pierce-Ward
```

:)

**Review by bluegenes — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -130,4 +130,4 @@ This software is under the AGPL license. Please see [LICENSE.txt](LICENSE.txt).
 Luiz Irber
 C. Titus Brown
 Mo Abuelanin
-N. Tessa Pierce
+N. Tessa Pierce-Ward
```
Comment (ctb) [inline README.md:133]: 👍

File: README.md [hunk 3]
```diff
@@ -121,4 +121,13 @@ maturin build
...
-CTB Aug 2023
+## License
+
+This software is under the AGPL license. Please see [LICENSE.txt](LICENSE.txt).
+
+## Authors
+
+Luiz Irber
+C. Titus Brown
+Mo Abuelanin
```
Comment (mr-eyes) [inline README.md:132]:
```suggestion
Mohamed Abuelanin
```

**Review by mr-eyes — APPROVED** [review]

**Comment by ctb** [comment]
🎉

**Pull request merged**